tests/entrypoints/openai/test_openai_schema.py

```diff
@@ -45,5 +48,13 @@ def get_schema(server):
 @schema.parametrize()
 @schema.override(headers={"Content-Type": "application/json"})
 async def test_openapi_stateless(case):
-    #No need to verify SSL certificate for localhost
-    await case.call_and_validate(verify=False)
+    key = (
+        case.operation.method.upper(),
+        case.operation.path,
+    )
+    timeout = {
+        ("POST", "/v1/chat/completions"): 30,
```

Comment:
this endpoint often times out for me. See https://schemathesis.readthedocs.io/en/stable/recipes.html#per-route-request-timeouts

Reply:
Odd for me it didn't happen

Reply:
might be because I'm using CPU platform. I'll revert.